test/backwards_compat/CMakeLists.txt

```
@@ -10,7 +10,7 @@ set_tests_properties(
 )
 
 
-if (${ROOT_VERSION} VERSION_GREATER_EQUAL 6.34)
+if (${ROOT_VERSION} VERSION_GREATER_EQUAL 6.34 AND ${podio_VERSION} VERSION_LESS_EQUAL 1.2)
```

```suggestion
if (${ROOT_VERSION} VERSION_GREATER_EQUAL 6.34 AND ${podio_VERSION} VERSION_GREATER_EQUAL 1.3)
```
I think this should be the proper way to do this for the future.